x-pack/plugins/observability_solution/observability/public/components/alerts_flyout/alerts_flyout_footer.tsx

```diff
@@ -48,23 +45,20 @@ export function AlertsFlyoutFooter({ alert, isInApp }: FlyoutProps & { isInApp:
             </EuiButton>
           </EuiFlexItem>
         )}
-
-        {!isAlertDetailsEnabledPerApp(alert, config) ? null : (
-          <EuiFlexItem grow={false}>
-            <EuiButton
-              data-test-subj="alertsFlyoutAlertDetailsButton"
-              fill
-              href={
-                prepend &&
-                prepend(paths.observability.alertDetails(alert.fields['kibana.alert.uuid']))
-              }
-            >
-              {i18n.translate('xpack.observability.alertsFlyout.alertsDetailsButtonText', {
-                defaultMessage: 'Alert details',
-              })}
-            </EuiButton>
-          </EuiFlexItem>
-        )}
+        <EuiFlexItem grow={false}>
```

Comment:
I think one of the problems we are solving with baseline alert detail pages is to have inconsistent UX. If we remove this button in flyout, we still have "..."->View alert details redirecting user to new alert details page. Then for some alerts, we see "Alert details" in flyout, for some we don't. But from another place, user can go to alert details page. I feel this is more confusing for users, so I would be in favor of keeping this button to have same functionality on all alerts.

But I see your point, it's going to be exact same content as flyout. Hopefully we implement dedicated alert detail pages soon.

Reply:
Yeah I agree it's a good thought, and I also vote to leave it for now. Hopefully near-term evolution of this will be:

(1) Merge this baseline alert details
(2) Make flyout for all alerts be the same tabbed view as the page, responsively fit to flyout
(3) Change this button to be something more like "View as page" or "Permalink" or something that makes sense there, we have this pattern other places, I think